simpleviewinc/re-theme
# postinstall-update

**Ticket**: n/a

## Context

* when importing this repo as an npm package, it will try to run the `postinstall` script. but it will fail because it doesn't exist
* we need to include `/scripts` in the `files` key
* also update the postisntall script to only run when it's not inside `node_modules'

## Goal

* postinstall script wont run if its installed as a node module, but should run if its cloned as a repo

## Updates

* `package.json`
    * updated postinstall cmd
* added PR template
* added a check to not run postinstall if current dir is in `node_modules`

## Testing
* pull this branch
* run `yarn install` || `yarn postinstall`
* verify that it runs the script
     * you should see msg 
```
-----git update assume-unchange folder: build-----
-----running yarn app:install-----
```

* delete the `/scripts` folder entirely
* run `yarn postinstall`
* verify that it does't throw error
